src/linked_list_cycle/main.cpp

```diff
@@ -16,24 +16,24 @@ struct ListNode{
 bool hasCycle(ListNode* head){
 	ListNode* fast = head;
 	ListNode* slow = head;
-	while(fast){
-		if(!fast->next) return false;
+	while (fast) {
+		if (!fast->next)  return false;
 		fast = fast->next->next;
 		slow = slow->next;
-		if(fast == slow) {
+		if (fast == slow) {
 			return true;
 		}
 	}
 	return false;
 }
 
-//测试只有一个结点，结果应要为无环
+/*测试只有一个结点，结果应要为无环*/
 TEST(hasCycle,case1){
 	ListNode* head = new ListNode(0);
 	ASSERT_EQ(hasCycle(head), false) << "single element should has no loop";
 }
 
-//测试有六个结点的无环链表
+/*测试有六个结点的无环链表*/
```

Comment:
You can use either the // or the /* */ syntax; however, // is much more common. Be consistent with how you comment and what style you use where.